pkg/virt-launcher/virtwrap/nichotplug_test.go

```diff
@@ -0,0 +1,166 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2023 Red Hat, Inc.
+ *
+ */
+
+package virtwrap
+
+import (
+	"fmt"
+
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	v1 "kubevirt.io/api/core/v1"
+
+	"kubevirt.io/kubevirt/pkg/network/vmispec"
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/cli"
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/converter"
+)
+
+var _ = Describe("nic hotplug on virt-launcher", func() {
+	const (
+		nadName     = "n1n"
+		networkName = "n1"
+	)
+
+	DescribeTable(
+		"hotplugVirtioInterface SUCCEEDS for",
+		func(vmi *v1.VirtualMachineInstance, domain *api.Domain, result libvirtClientResult) {
+			vim := newVirtIOInterfaceManager(
+				converterContext(),
+				mockLibvirtClient(gomock.NewController(GinkgoT()), result),
+				&fakeVMConfigurator{},
+			)
+			Expect(vim.hotplugVirtioInterface(vmi, domain)).To(Succeed())
+		},
+		Entry(
+			"VMI without networks, whose domain also doesn't have interfaces, does **not** attach any device",
+			&v1.VirtualMachineInstance{Spec: v1.VirtualMachineInstanceSpec{Networks: []v1.Network{}}},
+			dummyDomain(),
+			libvirtClientResult{expectedAttachedDevices: 0},
+		),
+		Entry("VMI with 1 network (with PodConfigDone: true), not present in the domain",
+			vmiWithSingleBridgeInterfaceWithPodInterfaceReady(networkName, nadName),
+			dummyDomain(),
+			libvirtClientResult{expectedAttachedDevices: 1},
+		),
+	)
+
+	DescribeTable(
+		"hotplugVirtioInterface FAILS when",
+		func(vmi *v1.VirtualMachineInstance, domain *api.Domain, configurator vmConfigurator, result libvirtClientResult) {
+			ctrl := gomock.NewController(GinkgoT())
```

Comment:
Why the previous test didn't need it?

Reply:
They have it, https://github.com/kubevirt/kubevirt/pull/9429/files#diff-64397cde5b13235bcdf9689428c2b97b1542a60845ca9ffff28d9d68565c9a51R47.

I guess I can inline both calls for consistency - this one had slipped me by.